pyproject.toml

```
@@ -1,33 +1,83 @@
 # For TOML reference
 # https://learnxinyminutes.com/docs/toml/
+[project]
+urls = { Documentation = "https://automl.github.io/DEHB/", Github = "https://github.com/automl/DEHB" }
+
+name = "DEHB"
+version = "0.7.0"
```

```suggestion
version = "0.0.7"
```
Version should be 0.0.7